### ISSUE TITLE
Filter out 'versions' of the song?

### ISSUE THREAD

**Issue by x-Misery** [jakubito/spotify-release-list#53]
Would be cool if there was an ability to remove sped up, slowed down, chopped, instrumental & acapella versions of the songs the same way you can filter remixes & various artists. Perhaps you can do it simply by checking the title?

**Comment by jakubito** [comment]
Yes, this could be solved by adding a new filter, matching any of these keywords:

- "instrumental"
- "acoustic"
- "acapella"

Feel free to add more to the list

**Comment by jakubito** [comment]
[v3.8.0](https://github.com/jakubito/spotify-release-list/releases/tag/v3.8.0)
